# Author: @maxim-masterov (SURF)
# This patch fixes the error with the multiple definitions of global variables.
diff -Nru ncl-6.6.2.orig/ni/src/ncl/NclApi.c ncl-6.6.2/ni/src/ncl/NclApi.c
--- ncl-6.6.2.orig/ni/src/ncl/NclApi.c	2021-08-23 11:16:17.620033156 +0200
+++ ncl-6.6.2/ni/src/ncl/NclApi.c	2021-08-25 18:15:32.792728681 +0200
@@ -140,9 +140,9 @@
 extern char *the_input_buffer_ptr;
 extern int the_input_buffer_size;
 
-FILE *thefptr;
-FILE *theoptr;
-int cmd_line;
+extern FILE *thefptr;
+extern FILE *theoptr;
+extern int cmd_line;
 extern int cur_line_number;
 extern char *cur_line_text;
 extern int cur_line_maxsize;
diff -Nru ncl-6.6.2.orig/ni/src/ncl/NclNewHDF5.c ncl-6.6.2/ni/src/ncl/NclNewHDF5.c
--- ncl-6.6.2.orig/ni/src/ncl/NclNewHDF5.c	2021-08-23 11:16:18.450026586 +0200
+++ ncl-6.6.2/ni/src/ncl/NclNewHDF5.c	2021-08-25 18:12:58.378953245 +0200
@@ -82,7 +82,7 @@
 
 #define NUMPOSDIMNAMES	6
 
-NclQuark possibleDimNames[NUMPOSDIMNAMES];
+extern NclQuark possibleDimNames[NUMPOSDIMNAMES];
 
 #ifndef FALSE
 #define FALSE           0
